src/test/CMakeLists.txt

```diff
@@ -356,8 +356,9 @@ target_link_libraries(ceph_multi_stress_watch librados global radostest
 add_executable(ceph_perf_local 
   perf_local.cc
   perf_helper.cc)
-#INTEL_SSE & INTEL_SSE2 flags
-set(PERF_LOCAL_FLAGS "-msse -msse2")
+
+include(CheckCompileTargets)
+set(PERF_LOCAL_FLAGS ${COMPILE_TARGETS_FLAGS})
```

Comment:
@cbodley Only SSE3 and NEON flags will be checked here. The update will be ready soon.